config/locales/en.yml

```diff
@@ -3452,6 +3457,8 @@ en:
   notice_successful_delete: "Successful deletion."
   notice_successful_cancel: "Successful cancellation."
   notice_successful_update: "Successful update."
+  notice_unsuccessful_create: "Creation failed."
+  notice_unsuccessful_create_with_reason: "Creation failed: %{reason}"
   notice_unsuccessful_update: "Update failed."
   notice_unsuccessful_update_with_reason: "Update failed: %{reason}"
```

Comment:
Not sure about the naming here - these could also be `error_unsuccessful_*` to reflect that they will usually be used with the `error` flash key.

Reply:
Leaning towards them being errors, but as we already have unsuccessful_update afterwards, it seems to be inconsistent already. If we do that, we could do it separately for both sets of keys